bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/communication/RotelConnector.java

```diff
@@ -88,15 +92,32 @@ protected void setConnected(boolean connected) {
      */
     public abstract void close();
 
+    protected void startReaderThread() {
+        Thread thread = readerThread;
+        if (thread == null || thread.isInterrupted()) {
+            thread = new RotelReaderThread(this, protocolHandler, readerThreadName);
+            readerThread = thread;
+            thread.start();
+        }
+    }
+
+    protected void stopReaderThread() {
+        Thread thread = readerThread;
+        if (thread != null) {
+            thread.interrupt();
+            try {
+                thread.join();
+            } catch (InterruptedException e) {
+            }
```

Comment:
Can you add comment here why the exception is ignored?

Reply:
Or even just:
```java
// Ignored
```

I don't remember if SAT complains when catch is empty.

Reply:
This is a general pattern you can find in many bindings. We request the interruption of a thread and then wait for this thread to be either completed or interrupted. So this is expected to catch InterruptedException. If you really want a comment, I can add something like "The thread is now interrupted. " but I do not see the value as it is relatively obvious.

Reply:
OK.